src/examples/DataCenter/Input/paraCfgCurrent.txt

```diff
@@ -1,20 +1,17 @@
 ParameterName	ParameterValue
 environmentFileName	DataCenter.xml
-generalizedScenario	1
+generalizedScenario	0
 indoorSwitch	1
 mobilitySwitch	0
 mobilityType	1
 numberOfNodes	2
-numberOfTimeDivisions	15
-referencePoint	[3,3,2]
+numberOfTimeDivisions	1
+referencePoint	[0,0,0]
 selectPlanesByDist	0
-switchmodelRray	1
-switchDGenerator	1
-switchRGenerator	1
 switchQDGenerator	1
 switchRandomization	0
 totalNumberOfReflections	2
-totalTimeDuration	15
+totalTimeDuration	14
```

Comment:
if you only consider 1 time division it doesn't make sense to set the duration to 14 seconds. as i also wrote in a previous commit, if you want to keep these parameters could you please check whether the position is chosen randomly or as the first line of the NodePosition file?

Reply:
check above reply